Participant import rework (#3688)
OpenSlides/openslides-backend: docs/actions/preface_special_imports.md

                // property name and type must match an entry in the given `headers`
                [property: string]: (boolean | number | string | date | object) []; // if is_list is set in corresponding header column, we need here also a list. `object` only on error
            };

>             list_deletions: { [string]: int } // optional and only used in imports that support it (no changes from the import even if there were changes due to changed database data)

Reviewer: ```suggestion
            list_deletions: { [string]: int } // optional and only used in imports that support change detection. See more in section 'Change Detection'.
```
TBH I don't really get this part: `(no changes from the import even if there were changes due to changed database data)`

Author: The `(no changes from the import even if there were changes due to changed database data)`-part is because it's not recalculating CD after import. With your previous change though, I'd agree, simply pointing to the description section should be enough.

        }[];  // row-list: Empty list, if `import` in payload was `false` to delete the import_preview-record on database
    }[[]];  // nested lists for actions and data row per action
}

```

## Change Detection
### The `changed` and `list_deletions` keywords

> These keywords are used in the preview to signify the relation between the cell content and the related database data.

Reviewer: ```suggestion
These keywords are used in the preview to signify the relation between the cell content and the related database data at the time of upload.
```

Any single field that contains a value that would change database data (i.e. by filling in a new field, changing a value or creating a new model) will be marked with `"changed": True`.

For list columns, every new element is marked with `"changed": True`. Any removed values are counted and then added to the `list_deletions`-dict with the import field name as keyword.

> These keywords are only written for update or reference rows in imports that support change detection, and if `changed` isn't written there it is assumed to be `False`.

Reviewer: When and why is this set to `False` explicitly?

Author: For the `default_password` field in case of `saml_id` being set for the account. The participant import creates an object with an empty-string value and a warning to alert the client that the field will be ignored because of the `saml_id`. In order to stop the change detection from recognizing the field as changed, `changed` is set to `False` explicitly (the automatic CD is written to only calculate for fields that don't have `changed` set manually).
